### DISCUSSION TITLE
Changing the build command from the CLI

### DISCUSSION THREAD

**Discussion by kyriacos** [vercel/vercel#8484] [Help]
I just tried changing the build command through the UI on the website and the form input is broken (any keypress gives `[Object object]`). 
I submitted a bug report to support for that.

So i thought maybe i can do that through the CLI but i can't seem to find it. Not sure if it's there basically. I can see the env variables being pulled in and set in `.vercel/project.json` but changing those doesn't affect anything remotely and there is no push command. Is there another way?

If not can please feel free to change this to a feature request.

Thanks

**Comment by swarnava** [comment]
Hi,

Sorry about the trouble. Can you try now and how it goes?
